### PR TITLE
fix: dune build after #36

### PR DESCRIPTION
**Pull request by anmonteiro**
fixes the issue reported in https://github.com/xavierleroy/cryptokit/pull/36#discussion_r1667748590

### CODE REVIEW

File: src/aesni.c [hunk 1]
```diff
@@ -301,7 +301,7 @@ EXPORT void aesniDecrypt(const unsigned char * key, int nrounds,
   
 #else
 
-int aesni_available = 0;
+EXPORT int aesni_available = 0;
```
Comment (anmonteiro) [inline src/aesni.c:304]: this was `EXPORT`ed in the `#ifdef` branch but not the `#else` branch.

File: src/dune [hunk 2]
```diff
@@ -26,7 +26,27 @@
          stubs-ghash
          stubs-poly1305
          stubs-siphash
-         stubs-blake3))
+         stubs-blake3)
+  (extra_deps
```
Comment (anmonteiro) [inline src/dune:30]: you can check that these are now included in the build but not compiled, e.g. with

```
$ dune build -p cryptokit --display=short

```

**Comment by xavierleroy** [comment]
Looks promising!  I'll review tomorrow.  Thanks a million for your help!

**Review by xavierleroy — APPROVED** [review]
Works great, thanks!

**Pull request merged**